configs/_base_/datasets/vaihingen.py

```diff
@@ -0,0 +1,49 @@
+# dataset settings
+dataset_type = 'VaihingenDataset'
```

Comment:
Might be redundant.
https://github.com/open-mmlab/mmsegmentation/pull/1171/files